sdcm/nemesis.py

```diff
@@ -6409,3 +6456,11 @@ class DisableBinaryGossipExecuteMajorCompaction(Nemesis):
 
     def disrupt(self):
         self.disrupt_disable_binary_gossip_execute_major_compaction()
+
+
+class EdquotNemesis(Nemesis):
```

Comment:
Name of the class should be similar to the name of the distrupt function
Edquot is clear or known enough, from my POV.

The function name is clearer

Reply:
I haven't fully got you. Should I change the `disrupt_end_of_quota_nemesis` to `disrupt_edquot_nemesis` you mean ?

Reply:
I think the other way around `EndOfQoutaNemesis` has more meaning to me then `EdquotNemesis` sounds like a typo at first look.

yeah, I know we have `EnospcNemesis`, but that name was like that before I joined scylla...
`NoSpaceNemesis` would have been more readable (especially for newcomers)

Reply:
I totally agree with you. 
I've used the `EDQUOT` because that's how it is described [here](https://github.com/scylladb/qa-tasks/issues/1086) and [here](https://github.com/scylladb/scylladb/pull/12653)
I guess its a known term in Linux.
Whatever you decide, no problem to change this tiny thing.